software/base/src/test/java/org/apache/brooklyn/entity/brooklynnode/BrooklynNodeTest.java

```diff
@@ -120,18 +132,58 @@ public void run() {
             }
         });
     }
-    
+
 
     @Test
     public void testCanStartSameNode() throws Exception {
         // not very interesting as do not have REST when run in this project
         // but test BrooklynNodeRestTest in downstream project does
         BrooklynNode bn = app.createAndManageChild(EntitySpec.create(BrooklynNode.class, SameBrooklynNodeImpl.class));
         bn.start(MutableSet.<Location>of());
-        
-        Assert.assertEquals(bn.getAttribute(Attributes.SERVICE_UP), (Boolean)true);
+
+        Assert.assertEquals(bn.getAttribute(Attributes.SERVICE_UP), (Boolean) true);
         // no URI
         Assert.assertNull(bn.getAttribute(BrooklynNode.WEB_CONSOLE_URI));
     }
 
+    @Test(groups = "Integration")
+    public void testOpenHttpConsoleWebPort() throws Exception {
+
+        BrooklynNodeWitOpenedPorts brooklynNode = app.createAndManageChild(
+                EntitySpec.create(BrooklynNodeWitOpenedPorts.class)
+                        .configure(BrooklynNode.ON_EXISTING_PROPERTIES_FILE, BrooklynNode.ExistingFileBehaviour.DO_NOT_USE)
+                        .configure(BrooklynNode.NO_WEB_CONSOLE_AUTHENTICATION, true)
+                        .configure(BrooklynNode.HTTP_PORT, PortRanges.fromString("8082+")));
+
+        app.start(ImmutableList.of(loc));
+        assertNotNull(app);
+
+        log.info("started " + app + " containing " + brooklynNode + " for " + JavaClassNames.niceClassAndMethod());
+
+        //EntityTestUtils.assertAttributeEqualsEventually(brooklynNode, BrooklynNode.SERVICE_UP, true);
```

Comment:
Why commented out? Delete this line, as it is replaced by the line below it I presume.

General note: never leave commented out code, unless there is a very clear comment with it that says what it is for and under what circumstances one would want to uncomment it.